src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -22,10 +24,18 @@ export class LearningObjectsComponent {
   activeLearningObject;
   adminStatusList =  Object.keys(LearningObject.Status);
   selectedStatus: string;
+  sub: Subscription;
```

Comment:
What is the purpose of this variable? Why not jut subscribe and execute the callback function?